indico/modules/auth/client/js/signup.jsx

```diff
@@ -141,6 +142,7 @@ function Signup({
               <SyncedFinalInput
                 name="affiliation"
                 label={Translate.string('Affiliation')}
+                required={ moderated && mandatoryFields.includes("Affiliation") }
```

Comment:
please make sure to use eslint so it fixes formatting issues (through its *prettier* integration)